cmd/web-handlers.go

```diff
@@ -798,7 +845,6 @@ func (web *webAPIHandlers) SetBucketPolicy(r *http.Request, args *SetBucketPolic
 		}
 		return toJSONError(err, args.BucketName)
 	}
-	reply.UIVersion = browser.UIVersion
```

Comment:
Sending a UIVersion is important for successful operation. otherwise it won't work.